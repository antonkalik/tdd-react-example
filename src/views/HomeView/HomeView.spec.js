import { render, screen } from "@testing-library/react";
import { HomeView } from "./index";

describe("<HomeView />", () => {
  it("should render view", () => {
    render(<HomeView />);
    expect(screen.getByTestId("home-view")).toMatchSnapshot();
    expect(screen.getByTestId("home-view")).toHaveTextContent("Home View");
  });
});
