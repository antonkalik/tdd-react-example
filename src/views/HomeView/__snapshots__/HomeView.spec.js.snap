// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomeView /> should render view 1`] = `
<div
  data-testid="home-view"
>
  <h1>
    Home View
  </h1>
</div>
`;
